test/invoice_item_repository_test.rb

```diff
@@ -5,7 +5,7 @@
 class InvoiceItemRepositoryTest < MiniTest::Test
   attr_reader :invoice_items_repo
   def setup
-    @invoice_items_repo = InvoiceItemRepository.new('test/support/sample_invoice_items.csv')
+    @invoice_items_repo = InvoiceItemRepository.new('test/support/invoice_items.csv')
```

Comment:
Line is too long. [85/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.